notebooks/data-sources/oc-github-repo/github_PR_EDA.ipynb

```diff
@@ -2,17 +2,19 @@
  "cells": [
```

Comment:
<p>nit: is there a reason there <code>knobunc</code>  has a comment after username? Not necessary to remove if so, but just a bit out of place :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>I think @chauhankaranraj followed the same structure from the OWNERS file here https://github.com/openshift/origin/blob/master/OWNERS. :) Correct me if I'm wrong @chauhankaranraj</p>